# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/xfsprogs/libuuid.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./configure.in.orig	2003-03-14 15:01:16.000000000 +0100
+++ ./configure.in	2003-03-14 15:01:59.000000000 +0100
@@ -185,11 +185,11 @@
 dnl
 AC_ARG_ENABLE(shared-uuid,
 	[ --enable-shared-uuid=[yes/no]	Link shared libuuid [default=no].],
-	libuuid="/usr/lib/libuuid.a"
+	libuuid="/lib/libuuid.a"
 	if test "$enable_shared_uuid" = yes; then
 		libuuid="-luuid"
 	fi,
-	libuuid="/usr/lib/libuuid.a")
+	libuuid="/lib/libuuid.a")
 AC_SUBST(libuuid)
 
 dnl Check if we have a type for the pointer's size integer (__psint_t)
